<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-OHBQ9CF" failures="2" tests="5" name="Test" time="28.241" errors="0" timestamp="2023-03-27T15:36:03 IST">
  <testcase classname="Test.scenario_01_Test" name="validate_Entered_Message" time="1.507"/>
  <testcase classname="Test.scenario_01_Test" name="validate_Simple_Form_Demo_URL" time="0.581"/>
  <testcase classname="Test.scenario_03_Test" name="validate_Error_Message" time="0.992">
    <failure type="java.lang.AssertionError" message="expected [KDGPlease fill out this field.] but found [Please fill out this field.]">
      <![CDATA[java.lang.AssertionError: expected [KDGPlease fill out this field.] but found [Please fill out this field.]
at Test.scenario_03_Test.validate_Error_Message(scenario_03_Test.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validate_Error_Message -->
  <testcase classname="Test.scenario_03_Test" name="validate_Success_Message" time="5.201"/>
  <testcase classname="Test.scenario_02_Test" name="validate_Dragged_Value" time="4.375">
    <failure type="java.lang.AssertionError" message="expected [59] but found [95]">
      <![CDATA[java.lang.AssertionError: expected [59] but found [95]
at Test.scenario_02_Test.validate_Dragged_Value(scenario_02_Test.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validate_Dragged_Value -->
</testsuite> <!-- Test -->
